scenarios/preset/helpers/neofs_cli.py

```diff
@@ -13,18 +13,17 @@ def create_container(endpoint, policy, wallet_file, wallet_config):
         print(f" > Container has not been created:\n{output}")
         return False
     else:
-        try:
-            fst_str = output.split('\n')[0]
-        except Exception:
-            print(f"Got empty output: {output}")
-            return False
-        splitted = fst_str.split(": ")
-        if len(splitted) != 2:
-            raise ValueError(f"no CID was parsed from command output: \t{fst_str}")
-
-        print(f"Created container: {splitted[1]}")
-
-        return splitted[1]
+        # Regular expression to find the container ID, case-insensitive
+        pattern = r"container ID: ([A-Za-z0-9]+)"
+
+        # Search for the pattern in the output text with case-insensitive flag
+        match = re.search(pattern, output, re.IGNORECASE)
```

Comment:
Why `IGNORECASE`? You have `A-Za-z` anyway.

Reply:
That's because "container ID" can be written in any case. The `re.IGNORECASE` flag ensures that the search is case-insensitive, allowing "container ID:" to match any case variation in the input text.